recipes/snakemake/LICENSE.md

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2016 Johannes Köster <johannes.koester@tu-dortmund.de>
```

Comment:
I've [made a PR upstream](https://bitbucket.org/snakemake/snakemake/pull-requests/375/add-licensemd-to-manifestin/diff) to include this in the PyPI tarball.